corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -111,6 +111,20 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
         }
     }
 
+    function getMatchingStyles(pattern, styleStr) {
+        let retVal = [];
```

Comment:
`matchingStyles` might be a better name.

Reply:
My preference is for names of non-boolean functions to be verbs so I would prefer it as is over `matchingStyles`

Reply:
It's just that retVal does not tell you anything about the value stored in it.

Reply:
Oh my bad, I thought you were referring to the function name. Yes, I agree with you about `retVal`[1c8f778](https://github.com/dimagi/commcare-hq/pull/33333/commits/1c8f778e0a8c6054413f447d3eaed95cb97940b1)